content/applications/productivity/calendar.rst

```diff
@@ -1,9 +1,285 @@
-:nosearch:
+:show-content:
 
 ========
 Calendar
 ========
 
+.. toctree::
+   :titlesonly:
```

Comment:
Can remove this TOC text since you already have defined the TOC at the bottom of the file :)
```suggestion
```